gson/src/test/java/com/google/gson/functional/StringTest.java

```diff
@@ -149,6 +149,7 @@ public void testAssignmentCharSerialization() {
    * Created in response to http://groups.google.com/group/google-gson/browse_thread/thread/2431d4a3d0d6cb23
    */
   @Test
+  @SuppressWarnings("UnicodeEscape")
```

Comment:
I think this is actually a bug with the test below. As reported by Error Prone `\u003d` (with a single backslash) in the string below is replaced by the Java Compiler, so this Unicode escape is not processed by Gson.

It should probably have two backslashes: `\\u003d`.

Reply:
Oh wow, good catch!